spp_service_points/tests/test_service_point.py

```diff
@@ -85,7 +85,7 @@ def setUpClass(cls):
     #     self.assertNotEqual(
     #         self.service_point_1.phone_sanitized,
     #         False,
-    #         "Service Point 1 having correct Iraq phone number format, should have phone sanitized!",
+    #         "Service Point 1 having correct phone number format, should have phone sanitized!",
```

Comment:
@gonzalesedwin1123 I don't understand this assertion message, as the use of "correct" is confusing me.

Reply:
@kneckinator After reading the code I am now not sure of what to put in the assertion message. Should it be "Service Point 1 have a correct phone number format but should be sanitized"?

Reply:
I have no idea, either. :)

Reply:
Let's leave it for now.